src/components/Icons/Essencials/Flag.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function Flag({
+  id = 'FlagIcon',
+  color = colors.space100,
+  viewBox = getViewBox(),
+  width = getSize(),
+  height = getSize(),
+  ...props
+}: IconProps) {
+  const fill = React.useMemo(() => getFill({ gradient: props.gradient, color, id }), [color, props.gradient, id]);
+
+  return (
+    <BaseIcon id={id} color={color} width={width} height={height} viewBox={viewBox} {...props}>
+      <Path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
🔧 Passou batido esses aqui
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

Reply:
comment resolved in https://github.com/magnetis/astro-native/pull/574/commits/897149847831d1ec7f69709265117ecc9502761a